lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -40,19 +40,19 @@ class WP_Font_Collection {
 	 */
 	public function __construct( $config ) {
 		if ( empty( $config ) || ! is_array( $config ) ) {
-			throw new Exception( 'Font Collection config options is required as a non-empty array.' );
+			throw new Exception( __( 'Font Collection config options is required as a non-empty array.', 'gutenberg' ) );
```

Comment:
I'm a little unsure whether the text when throwing an exception should also be translatable.